src/components/CountdownTimer/CountdownTimer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { TimerList } from './components/TimerList';
+import { TimerTitle } from './components/TimerTitle';
+import './CountdownTimer.scss';
+
+interface CountdownTimerProps {
+  launchDate: number;
+}
+
+type CalcTimerValueFn = (
+  setFunc: (value: number) => void,
+  divider: number,
+  module?: number,
+) => void;
+
+type CalculateValuesFn = () => void;
+
+const MILLISEC_IN_DAY = 1000 * 60 * 60 * 24;
+const MILLISEC_IN_HOUR = 1000 * 60 * 60;
+const MILLISEC_IN_MIN = 1000 * 60;
+const MILLISEC_IN_SEC = 1000;
+const HOURS_IN_DAY = 24;
+const MIN_IN_HOUR = 60;
+const SEC_IN_MIN = 60;
```

Comment:
do not calculate this values and move to enum

Reply:
done